iiif_prezi3/base.py

```diff
@@ -54,9 +56,9 @@ def jsonld(self, **kwargs):
         pydantic_args = ["include", "exclude", "by_alias", "encoder"]
         dict_kwargs = dict([(arg, kwargs[arg]) for arg in kwargs.keys() if arg in pydantic_args])
         json_kwargs = dict([(arg, kwargs[arg]) for arg in kwargs.keys() if arg not in pydantic_args])
-        return json.dumps({"@context": "http://iiif.io/api/presentation/3/context.json", **self.dict(exclude_unset=False, exclude_defaults=False, exclude_none=True, **dict_kwargs)}, **json_kwargs)
+        return json.dumps({"@context": "http://iiif.io/api/presentation/3/context.json", **self.dict(exclude_unset=False, exclude_defaults=False, exclude_none=True, by_alias=True, **dict_kwargs)}, **json_kwargs)
```

Comment:
This also requires a small change to the `pydantic_args` line (and also made me notice a bug in the current handling) - both fixed in #96